README.md

```diff
@@ -74,7 +74,7 @@ Repository  |   Description |   License |
 Repository  |   Description |   License |
 |---|---|---|
 |   [Async-JavaScript](https://github.com/frontarm/async-javascript-cheatsheet)  |    Cheatsheet for promises and async/await.    |   [MIT](https://github.com/frontarm/async-javascript-cheatsheet/blob/master/LICENSE.md)  |
-|   [Bootstrap 5](https://github.com/themeselection/bootstrap-cheatsheet)  |  An interactive list of Bootstrap 5 classes, variables, and mixins  |  [MIT](https://github.com/tiimgreen/github-cheat-sheet/blob/master/LICENSE ) |
+|   [Bootstrap 5](https://github.com/themeselection/bootstrap-cheatsheet)  |  An interactive list of Bootstrap 5 classes, variables, and mixins  |  [MIT](https://github.com/themeselection/bootstrap-cheatsheet ) |
```

Comment:
```suggestion
|   [Bootstrap 5](https://github.com/themeselection/bootstrap-cheatsheet)  |  An interactive list of Bootstrap 5 classes, variables, and mixins  |  [MIT](https://github.com/themeselection/bootstrap-cheatsheet/blob/master/LICENSE.md ) |
```

Reply:
Ok. I'm closing this pull request and will open the new one with the suggested changes.
Thank you.🙂